### PR TITLE
Otlp exporter documentation update

### PR DESCRIPTION
**Pull request by dszmigielski**
## Changes

Adding documentation for new env variables support added [here](https://github.com/open-telemetry/opentelemetry-dotnet/pull/2188).

For significant contributions please make sure you have completed the following items:

* [ ] `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md [hunk 1]
```diff
@@ -32,6 +32,11 @@ You can configure the `OtlpExporter` through `OtlpExporterOptions` properties:
 * `BatchExportProcessorOptions`: Configuration options for the batch exporter.
   Only used if ExportProcessorType is set to Batch.
 
+Some of these can also be set by environment variables following [specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md). For now we support following variables:
+* `OTEL_EXPORTER_OTLP_ENDPOINT`
+* `OTEL_EXPORTER_OTLP_HEADERS`
+* `OTEL_EXPORTER_OTLP_TIMEOUT`
+
```
Comment (pellared) [inline src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md:39]: Can you make the README structure similar to: https://github.com/open-telemetry/opentelemetry-dotnet/blob/5bdebf90039d09dbc5efd5073441b9172f247e58/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md ?

Comment (dszmigielski) [reply src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md]: Done

File: src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md [hunk 2]
```diff
@@ -35,6 +37,17 @@ You can configure the `OtlpExporter` through `OtlpExporterOptions` properties:
...
+## Environment Variables
+
+The following environment variables can be used to override the default
+values of the `OtlpExporterOptions` (See [specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md)).
+
+| Environment variable          | `OtlpExporterOptions` property |
+| ------------------------------| -------------------------------|
+| `OTEL_EXPORTER_OTLP_ENDPOINT` | `Endpoint`                     |
+| `OTEL_EXPORTER_OTLP_HEADERS`  | `Headers`                      |
+| `OTEL_EXPORTER_OTLP_TIMEOUT`  | `TimeoutMilliseconds`          |
```
Comment (pellared) [inline src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md:49]: `OTEL_EXPORTER_OTLP_TIMEOUT` has a different unit - seconds - am I correct?

If so I would give a note on that.

File: src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md [hunk 3]
```diff
@@ -35,6 +37,17 @@ You can configure the `OtlpExporter` through `OtlpExporterOptions` properties:
 See the [`TestOtlpExporter.cs`](../../examples/Console/TestOtlpExporter.cs) for
 an example of how to use the exporter.
 
+## Environment Variables
+
+The following environment variables can be used to override the default
+values of the `OtlpExporterOptions` (See [specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md)).
```
Comment (pellared) [inline src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md:43]: regarding
> (See [specification...

I suggest putting 

```
following the [OpenTelemetry specification]https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md).
```

After 

> The OTLP (OpenTelemetry Protocol) exporter communicates to an OpenTelemetry Collector through a gRPC protocol.

File: src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md (hunk 2 above)
Comment (dszmigielski) [reply src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md:49]: Actually the specification for this variable does not explicitly say what unit is used.

**Review by pellared — APPROVED** [review]

File: src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md (hunk 2 above)
Comment (pellared) [reply src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md:52]: I see. LGTM then

**Review by cijothomas — APPROVED** [review]

**Pull request merged**